TransCore/RPGCode.xml

```diff
@@ -2086,15 +2086,15 @@
 								{
 									label: (or (@ pageDesc 'acceptLabel)
 										(msnTranslate (@ options 'missionObj) 'AcceptLabel)
-										(typTranslate &dsRPGMission; 'actionAccept)
+										(typTranslate &dsDockScreenBase; 'actionAccept)
 										)
 									code: (@ pageDesc 'acceptCode)
 									nextPage: 'missionAccept
 									}
 								{
 									label: (or (@ pageDesc 'declineLabel)
 										(msnTranslate (@ options 'missionObj) 'DeclineLabel)
-										(typTranslate &dsRPGMission; 'actionDecline)
+										(typTranslate &dsDockScreenBase; 'actionDecline)
 										)
```

Comment:
Well, rpgPagePaneInit is mostly used by dsRPGMission, but it can be used by other screens, and is in a couple cases, which is why it's not in RPGMissions.xml, I guess. It doesn't refer to dsRPGMission anywhere else, and already had 5 other calls to (typTranslate &dsDockScreenBase...), so I think this is better.